### PR TITLE
Bump foreman_ansible to 1.5.2

### PR DESCRIPTION
**Pull request by dLobatog**
* [x] 1.16

### CODE REVIEW

**Comment by dLobatog** [comment]
This depends on https://github.com/theforeman/foreman-packaging/pull/2255

**Comment by dLobatog** [comment]
[test]
[test packaging]

**Comment by ekohl** [comment]
[test packaging]

File: rubygem-foreman_ansible/rubygem-foreman_ansible.spec [hunk 1]
```diff
@@ -19,21 +19,19 @@ Requires: %{?scl_prefix_ruby}ruby(release)
 Requires: %{?scl_prefix_ruby}rubygems
 Requires: %{?scl_prefix}rubygem(deface) < 2.0
 Requires: %{?scl_prefix}rubygem(foreman-tasks) >= 0.8
-Requires: %{?scl_prefix}rubygem(foreman-tasks) < 1.0
+Requires: %{?scl_prefix}rubygem(foreman-tasks) < 0.12
 Requires: %{?scl_prefix}rubygem(dynflow) >= 0.8.14
 Requires: %{?scl_prefix}rubygem(dynflow) < 0.9.0
-Requires: %{?scl_prefix}rubygem(foreman_ansible_core) >= 1.0
-Requires: %{?scl_prefix}rubygem(foreman_ansible_core) < 2.0
+Requires: %{?scl_prefix}rubygem(foreman_ansible_core) > 2.0.2
```
Comment (ekohl) [inline rubygem-foreman_ansible/rubygem-foreman_ansible.spec]: This line is breaking. I think you meant ```>=``` in the gemspec.

Comment (ekohl) [reply rubygem-foreman_ansible/rubygem-foreman_ansible.spec]: This line still needs an update. Looks like you did fix the BuildRequires.

**Comment by ekohl** [comment]
I'll let the test finish before merging.

**Comment by dLobatog** [comment]
@ekohl Test finished :smile:

**Comment by ekohl** [comment]
http://koji.katello.org/koji/taskinfo?taskID=87729

**Pull request merged**